### PR TITLE
[Takeover + engage page] SBI JP

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add SBI takeover
- Add SBI engage page
- Implement multiple takeovers rotation

## QA

- Go to [http://0.0.0.0:8012/](http://0.0.0.0:8012/) & [/engage/sbi](http://0.0.0.0:8012/engage/sbi)
- Compare to [brief-1](https://github.com/canonical-web-and-design/web-squad/files/3405659/fin_SBI.Bits.landing.page_IPR.docx), [brief-2](https://drive.google.com/drive/u/1/folders/1lJzHYRT9l4C0CnbpsWFzn09JIYOUT6Ze) and [design]https://github.com/canonical-web-and-design/web-squad/issues/1450)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1451

### CODE REVIEW

**Comment by solazio** [comment]
@SirSamTumless I need marketo information and return url for pdf

**Comment by anasereijo** [comment]
Please make the text 7 cols and image 5

File: templates/engage/sbi.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+      <p>SBI BITSは、日本の大手金融サービス企業であるSBIグループにITサービスとインフラを提供しています。SBIグループは250社以上、6,000人を超えるの従業員で構成されています。</p>
+      <p>製品化までの時間短縮とクライアントからの高い要求に対応するため、ベアメタルサーバーに代わるソリューションを模索していたSIB BITSは、OpenStackの採用を決めました。当初は既存のサプライヤーを検討しましたが、経済的で柔軟性のあるソリューションで本稼働に移行するために、外部サポートおよび専門性を備えたCanonicalを選択しました。</p>
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">CanonicalのOpenStackにはロックインのリスクがありません。アップストリームバージョンに近いため、こちらが望めばサポートの完全な内製化も可能です。</p>
+        <cite class="p-pull-quote__citation">SBI BITS社CIO、Georgi Georgiev氏</cite>
+      </blockquote>
+      <p>実装期間はわずか数週間ほどでしたが、費用対効果の高いOpenStackクラウドによる設備投資（CAPEX）および運用コスト（OPEX）の面で効果が現れており、SBI BITSはSBIグループ全体へのれプロいに自信を深めています。</p>
```
Comment (nobuto-m) [inline templates/engage/sbi.html]: "SBI BITSはSBIグループ全体へのれプロいに自信を深めています"

^^^ this sentence is broken.

I don't know the original text, but that's supposed to be like:

"SBI BITSはSBIグループ全体へのデプロイに自信を深めています"

Comment (solazio) [reply templates/engage/sbi.html]: @nobuto-m I pushed a commit to amend that sentence. Here is the [brief](https://github.com/canonical-web-and-design/web-squad/files/3405659/fin_SBI.Bits.landing.page_IPR.docx) if you want to double check the text, and also [the english version](https://docs.google.com/document/d/1ky7FmqiSEBi_fDMo3d2p7oeIrFhvSCCCJmohd6rc6uQ/edit)

Comment (nobuto-m) [reply templates/engage/sbi.html]: Looks good to me. Thanks!

**Comment by anasereijo** [comment]
LGTM

**Comment by SirSamTumless** [comment]
Upload the PDF to the assets:  [Canonical_SBi_0719.pdf](https://github.com/canonical-web-and-design/jp.ubuntu.com/files/3416832/Canonical_SBi_0719.pdf)
 
Link to it directly form the form below
<script src="//app-sjg.marketo.com/js/forms2/js/forms2.min.js"></script>
<form id="mktoForm_3428"></form>
<script>MktoForms2.loadForm("//app-sjg.marketo.com", "066-EOV-335", 3428);</script>

**Comment by SirSamTumless** [comment]
New ID:

 <script src="//app-sjg.marketo.com/js/forms2/js/forms2.min.js"></script>
<form id="mktoForm_3428"></form>
<script>MktoForms2.loadForm("//app-sjg.marketo.com", "066-EOV-335", 3428);</script>

**Pull request merged**